### PR TITLE
feat(cli): New CVE-ID parameter to 'lacework vuln ctr show cmd'

### PR DESCRIPTION
**Pull request by dmurray-lacework**
## Summary

Add new optional parameter to  'lacework vuln ctr show cmd' 
Usage:

```
❯ lacework vuln ctr show lacework vuln ctr show <sha> CVE-1234-56789             
                 CVE DETAILS                  
----------------------------------------------
    CVE ID            CVE-2021-24215          
    SEVERITY          Medium                  
    PACKAGE           example                 
    NAMESPACE         alpine:test             
    FEED              lacework                
    SRC                                       
    START TIME        2022-11-14T00:20:57Z    
    VERSION FORMAT    apk                     
    CURRENT VERSION   1.00.0-r1               
    FIX VERSION       1.20.00-r0              
    STATUS            VULNERABLE              
                                       
                          INTRODUCED IN LAYERS                          
------------------------------------------------------------------------
  COPY                                                                  
  dir:abcdefghjljk1234abcdef  
  in ./node_modules 
------------------------------------------------------------------------                                                    
  COPY                                                                  
  dir:abcdefghjljk1234abcdef  
  in ./node_modules 
------------------------------------------------------------------------                                                    
  COPY                                                                  
  dir:abcdefghjljk1234abcdef  
  in ./node_modules
------------------------------------------------------------------------                                                     
  COPY                                                                  
  dir:abcdefghjljk1234abcdef  
  in ./node_modules                                                     

```

## How did you test this change?

Tests
- [x] `TestBuildVulnContainerAssessmentReportsWithSingleCveFilter`
- [x] `TestBuildVulnContainerAssessmentReportsWithSingleCveFilterPackages`
- [x] `TestBuildVulnContainerAssessmentReportsWithSingleCveFilterJson`
- [x] `TestV2VulnerabilitiesFilterSingleVulnID`

Manual
`lacework vuln ctr show <sha> [cve_id]`
`lacework vuln ctr show <sha> [cve_id] --json`
`lacework vuln ctr show <sha> [cve_id] --packages`

## Issue

https://lacework.atlassian.net/browse/GROW-1440

### CODE REVIEW

**Review by afiune — CHANGES_REQUESTED** [review]

**Comment by afiune** [comment]
This is also a `fix:` not a `chore:`

File: api/v2_vulnerabilities_test.go [hunk 1]
```diff
@@ -80,6 +80,18 @@ func TestV2Vulnerabilities_Hosts_SearchAllPages_EmptyData(t *testing.T) {
 	assert.Equal(t, 0, len(response.Data))
 }
 
+func TestV2VulnerabilitiesFilterSingleVulnID(t *testing.T) {
+	var (
+		response api.VulnerabilitiesContainersResponse
+		vulnID   = "CVE-2017-12670"
+	)
+
+	json.Unmarshal([]byte(mockVulnerabilitiesContainersResponse()), &response)
```
Comment (afiune) [inline api/v2_vulnerabilities_test.go:89]: What if there are more than one vulnerability information about the same CVE id?

Comment (afiune) [reply api/v2_vulnerabilities_test.go:89]: I tried out "dirty image" that has this CVE ID four times and it doesn't really display the information correctly:
```
lacework vulnerability ctr show sha256:77b2d2246518044ef95e3dbd029e51dd477788e5bf8e278e418685aabc3fe28a CVE-2022-1271
```

Comment (dmurray-lacework) [reply api/v2_vulnerabilities_test.go:89]: I've updated this to repeat the table when the are multiple results of the same cve-id.

**Review by afiune — COMMENTED** [review]
I will come back and review this.

**Review by afiune — APPROVED** [review]
Let's just roll with it...

**Pull request merged**